src/formatting/expr.rs

```diff
@@ -444,7 +444,7 @@ pub(crate) fn format_expr(
     };
 
     expr_rw
-        .and_then(|expr_str| recover_comment_removed(expr_str, expr.span, context))
+        .map(|expr_str| recover_comment_removed(expr_str, expr.span, context))
```

Comment:
this was the one that confused me on my initial cursory glance as I thought both were `Option<String>`'s but see now that the signature on `recover_comment_removed` has been changed :+1:

Reply:
Yes, sorry, should have left a comment :/